docs/getting_started.rst

```diff
@@ -1,116 +1,44 @@
-
-  * add ``export CFME_TESTS_KEY="our shared key"`` into the activate script
-  * create ``.yaml_key`` file in project root containing the key
-
-
-* Ensure the following devel packages are installed (for building python dependencies):
-
-  * ``gcc``
-  * ``postgresql-devel``
-  * ``libxml2-devel``
-  * ``libxslt-devel``
-  * ``zeromq3-devel``
-  * ``libcurl-devel``
-  * ``redhat-rpm-config``
-  * ``gcc-c++``
-  * ``openssl-devel``
-  * ``libffi-devel``
-  * ``python-devel``
-  * Fedora (possibly RHEL-like systems) users:
-
-    * ``pkcon install gcc postgresql-devel libxml2-devel libxslt-devel zeromq3-devel libcurl-devel redhat-rpm-config gcc-c++ openssl-devel libffi-devel python-devel``
-
-    * On RHEL and derived systems, it will say the zeromq package is not available but that is ok.
-
-* Activate the virtual environment:
-
-To activate the virtualenv, the ``bin/activate`` script must be sourced. Bear in mind that you
-should have the two options added in the ``bin/activate`` script BEFORE you source it, otherwise it
-will not work.
-
-.. code-block:: bash
-
-   # Bash example:
-   cd /path/to/virtualenv
-   source bin/activate or . bin/activate
-
-* Install python dependencies:
-
-  * ``PYCURL_SSL_LIBRARY=nss pip install -Ur /path/to/virtualenv/cfme_tests/requirements.txt``
-  * If you get error from pycurl and you used this command, you might like to remove pycurl and try
-    installing it again with different SSL library set. The error message should give you an idea
-    what to try. For reinstallation, you will need to use the command mentioned in next bullet.
-  * If you forget to use the ``PYCURL_SSL_LIBRARY`` env variable and you get a pycurl error, you
-    have to run it like this to fix it:
-
-    * Ensure you have ``libcurl-devel`` installed (this was not a prerequisite before so it can
-      happen)
-    * Run ``PYCURL_SSL_LIBRARY=nss pip install -U -r requirements.txt --no-cache-dir``
-
-* run ``python scripts/disable-bytecode.py`` if you want to avoid having to clean up python bytecode
-* You copy/symlink the required YAML files into ``conf/`` if you have access to team's internal YAML
-  repository. Required YAML files are ``env``, ``cfme_data``, ``credentials``. If the file's
-  extension is ``.yaml`` it is loaded normally, if its extension is ``.eyaml`` then it is encrypted
-  and you need to have the decryption key in the ``cfme_tests/`` directory. You can also start them
-  from scratch by copying the templates in ``conf/`` and editing them to suit the environment you
-  use.
-* Set up a local selenium server that opens browser windows somewhere other than your
-  desktop. There is a Docker based solution for the browser, look at the script
-  ``scripts/dockerbot/sel_container.py``. That ensures you have the proper versions of browsers. You
-  can also set everything up in your system using Xvnc - :doc:`guides/vnc_selenium` .
 * Test! Run py.test. (This takes a long time, Ctrl-C will stop it)
```

Comment:
I thought there was supposed to be a wrapper component to this that we run the py.test by passing the args to the wrapper.